examples/web-instrumentation/browser/src/LoadTest.js

```diff
@@ -0,0 +1,148 @@
+import {CollectorExporter} from '@opentelemetry/exporter-collector';
+import {SimpleSpanProcessor} from '@opentelemetry/tracing';
+import {XMLHttpRequestPlugin} from '@opentelemetry/plugin-xml-http-request';
+import {ZoneScopeManager} from '@opentelemetry/scope-zone';
+import {WebTracerProvider} from '@opentelemetry/web';
+
+/**
+ * Class to run a load test and track reuqests.
+ */
+export class LoadTest {
+  /**
+   * Create a LoadTest
+   * @param {string} name - The name of the test
+   * @param {number} targetIter - The target number of iterations
+   * @param {number} delay - Time between requests
+   * @param {string} collectorURL - The URL of the OpenCensus agent
+   */
+  constructor(name, targetIter, delay, collectorURL) {
+    this.name = name;
+    this.targetIter = targetIter;
+    this.delay = delay;
+    this.buildId = 'v0.01';
+    this.numSent = 0;
+    this.numSuccess = 0;
+    this.numFail = 0;
+    const builder = new LogCollectorBuilder().setBuildId(this.buildId);
+    this.logCollector = builder.makeLogCollector();
+    this.logCollector.start(2000);
+    const webTracerWithZone = new WebTracerProvider({
+      // defaultAttributes: {'version': __VERSION__, 'test': name},
+      scopeManager: new ZoneScopeManager(),
+      plugins: [
+        new XMLHttpRequestPlugin({
+          ignoreUrls: ['/log', '/trace'],
+        }),
+      ],
+    });
+    const collectorOptions = {
+      url: collectorURL,
+    };
+    const exporter = new CollectorExporter(collectorOptions);
+    webTracerWithZone.addSpanProcessor(new SimpleSpanProcessor(exporter));
+  }
+
+  /**
+   * Send data to the server
+   * @param {object} data - The data to send
+   * @param {number} iterations - The number of iterations
+   * @param {number} interval - Time between each request
+   * @return {Observable} to subscribe to results of the test
+   */
+  runTest() {
+    let runDelay = 0.0;
+    for (let i = 0; i < this.targetIter; i++) {
+      runDelay += this.delay;
+      setTimeout(() => {
+        const data = new TestPayload(this.name);
+        this.sendData(data);
+      }, runDelay);
+    }
+    return new Observable( (subscriber) => {
+      setInterval(() => {
+        const results = this.getResults();
+        subscriber.next(results);
+        const numRecieved = this.numSuccess + this.numFail;
+        if (numRecieved === this.targetIter) {
+          subscriber.complete();
+        }
+      }, 1000);
+    });
+  }
+
+  /**
+   * Returns the test results
+   * @return {TestResults} test results
+   */
+  getResults() {
+    return new TestResults(
+        this.numSent,
+        this.numSuccess,
+        this.numFail,
+    );
+  }
+
+  /**
+   * Send data to the server
+   * @param {object} payload - The data to send
+   */
+  sendData(payload) {
+    const data = {name: payload.name,
+      reqId: payload.reqId,
+      tSent: payload.tSent};
+    this.numSent++;
+    let urlStr = '/data';
+    urlStr = `/data/${payload.reqId}`;
```

Comment:
Looks like above line is not required and can be replaced with const urlStr = `/data/${payload.reqId}`;

Reply:
Replaced as suggested